services/core-web/common/components/documents/ArchivedDocumentsSection.tsx

```diff
@@ -3,10 +3,11 @@ import DocumentTable from "@/components/common/DocumentTable";
 import { Typography } from "antd";
 import { DeleteOutlined } from "@ant-design/icons";
 import { Feature, isFeatureEnabled } from "@mds/common";
-import { MineDocument } from "@common/models/documents/document";
-
+import { MajorMineApplicationDocument } from "@common/models/documents/document";
+import { renderCategoryColumn } from "@/components/common/CoreTableCommonColumns";
+import * as Strings from "@common/constants/strings";
 interface ArchivedDocumentsSectionProps {
-  documents: MineDocument[];
+  documents: MajorMineApplicationDocument[];
```

Comment:
are we able to use the base model or does it not work that way?

Reply:
Can we instead pass in "file_location_type_code" as a prop like "categoryAttributeName", or use the category attribute in the document model? Currently we are not using the category attribute at all and I think this is a good use for it. Or pass in a prop like "extraColumns"? Otherwise, if that is not possible, and we are going to make this component be specific to MajorMineApplications we should rename it to reflect that.

Reply:
I have updated it. The required attributes are now in the base model. I did it this way to mirror the model in core-api.

Reply:
Can we change this to use the MineDocument though?